utils/biospecimen.js

```diff
@@ -191,20 +191,40 @@ const biospecimenAPIs = async (req, res) => {
         }
         const requestData = req.body;
         if(Object.keys(requestData).length === 0 ) return res.status(400).json(getResponseJSON('Request body is empty!', 400));
+        if(requestData['132929440']){
+            const boxId = requestData['132929440'];
+            const loginSite = requestData['789843387']
+            const { boxExists } = require('./firestore');
+            const exists = await boxExists(boxId, loginSite, requestData);
+            if (exists === true) return res.status(400).json(getResponseJSON('Box already exists!', 400));
+            if (exists === false) {
+                const { addBox } = require('./firestore');
+                await addBox(requestData);
+            }
+        }
+        return res.status(200).json({message: 'Addded!', code:200})
```

Comment:
I think there is a typo for the message 'Addded!'
 
Does the naming convention  for a status code of 200 have to be "Success!" or can it be anything that indicates a successful request?

Reply:
AFAIK its a message so can be anything. Anyways I changed it back to Success!